utils/generate-command-code.py

```diff
@@ -516,6 +516,9 @@ def fullname(self):
 
 
 def create_command(name, desc):
+    features = desc.get("feature_flags")
+    if features and len(features) and len([flag for flag in features if flag in args.feature_flag]) == 0:
+        return
```

Comment:
we already have a similar (different) mechanism for sentinel, see the SENTINEL and ONLY_SENTINEL command flags. i'd rather not introduce a new "system" (`feature_flags`) for that.

that is, if we don't take the first-party module approach discussed in the other comment.

Reply:
we can also use `ifdefs` like we did for cli_commands.c, maybe it plays more naturally with the current approach building one cluster or the other.
but either way, i'd like to avoid different content of commands.def depending on the build flags.

Reply:
Will switch to a sentinel style solution tomorrow